modules/op-creating-pipeline-templates-admin-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/working-with-pipelines-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="op-creating-pipeline-templates-admin-console_{context}"]
+= Creating pipeline templates in the Administrator perspective
+
+As a cluster administrator, you can create pipeline templates which developers can reuse when they create a pipeline on the cluster.
+
+.Prerequisites
+
+* You have access to an {product-title} cluster with cluster administrator permissions, and have switched to the *Administrator* perspective.
```

Comment:
"have switched" is the present perfect tense, while the simple past tense is preferred, or at least that's what I remember to be the case. "Tenses" in the IBM SG just talks of using the present and the simple past.

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense

Reply:
This sentence in my opinion would read very strangely if "have" is removed (presumably that's the suggestion?).

Sometimes there needs to be a little wiggle room on the tenses. For example, in OCP docs generally, we don't allow the use of "will", but in the IBM style guide one of the examples shows how sometimes this can't be avoided, so that the sentence reads properly:
![image](https://github.com/openshift/openshift-docs/assets/40172997/fb435b29-5c97-45a6-b536-ebde6110c9a8)

I would leave these as they are, especially since this PR has been open for over about 6 weeks now, and I'd prefer not to block merging on anything that isn't a serious issue or is merely a preference in terms of reviews.

Reply:
Agreed on this one